### PR TITLE
Vrf 209 add infra job spec script

### PR DESCRIPTION
**Pull request by vreff**

### CODE REVIEW

File: core/scripts/ocr2vrf/setup_ocr2vrf.go [hunk 1]
```diff
@@ -225,18 +225,242 @@ func setupOCR2VRFNodes(e helpers.Environment) {
...
+				"", // P2P Bootstrapper
+				e.ChainID,
+				dkgEncrypters[adjustedIndex],
+				dkgSigners[adjustedIndex],
+				*keyID,
+				dkgAddress.String(),
+				vrfCoordinatorAddress.String(),
+				*linkEthFeed,
+			)
+			fmt.Printf("VRF JOB FOR NODE %d:\n%v\n", i, vrfJob)
```
Comment (makramkd) [inline core/scripts/ocr2vrf/setup_ocr2vrf.go:414]: Lets make this a 0-based index for the non-bootstrap nodes to match our infra setup.

File: core/cmd/ocr2vrf_configure_commands.go [hunk 2]
```diff
@@ -76,17 +76,17 @@ KeyID                = "%s"
 SigningPublicKey     = "%s"
 `
 
-const ocr2vrfTemplate = `
+const OCR2vrfTemplate = `
```
Comment (makramkd) [inline core/cmd/ocr2vrf_configure_commands.go:79]:
```suggestion
const OCR2VRFTemplate = `
```

File: core/cmd/ocr2vrf_configure_commands.go [hunk 3]
```diff
@@ -393,12 +392,11 @@ func createDKGJob(lggr logger.Logger, app chainlink.Application, args dkgTemplat
 }
 
 func createOCR2VRFJob(lggr logger.Logger, app chainlink.Application, args ocr2vrfTemplateArgs) error {
-	sp := fmt.Sprintf(ocr2vrfTemplate,
+	sp := fmt.Sprintf(OCR2vrfTemplate,
```
Comment (makramkd) [inline core/cmd/ocr2vrf_configure_commands.go:395]:
```suggestion
	sp := fmt.Sprintf(OCR2VRFTemplate,
```

File: core/scripts/ocr2vrf/setup_ocr2vrf.go [hunk 4]
```diff
@@ -225,18 +235,243 @@ func setupOCR2VRFNodes(e helpers.Environment) {
...
+	fmt.Println("Deploying batch beacon consumer...")
+	loadTestConsumerAddress := deployLoadTestVRFBeaconCoordinatorConsumer(e, vrfCoordinatorAddress.String(), false, big.NewInt(*beaconPeriodBlocks))
+	addConsumer(e, vrfCoordinatorAddress.String(), loadTestConsumerAddress.String(), big.NewInt(int64(subID)))
+
+	for i := 0; i < *nodeCount; i++ {
+		// Apply forwarder args if using the forwarder.
+		if i > 0 {
+			adjustedIndex := i - 1
+			vrfJob := fmt.Sprintf(
+				cmd.OCR2vrfTemplate,
```
Comment (vreff) [inline core/scripts/ocr2vrf/setup_ocr2vrf.go:412]:
```suggestion
				cmd.OCR2VRFTemplate,
```

**Review by makramkd — APPROVED** [review]

**Pull request merged**